### PR TITLE
Fixes #1925 abbreviation related synonyms

### LINKED ISSUE #1925: [Typo/Bug] Fixes for abbreviations (narrow/broad)
Context: https://github.com/obophenotype/uberon/issues/2363#issuecomment-1446928901

**CL terms**
CL:0000037
CL:0000038
CL:0000040
CL:0000049
CL:0000051
CL:0000134
CL:0000145
CL:0000557
CL:0000595
CL:0000611
CL:0000798
CL:0000936
CL:0001059
CL:0001066
CL:0002066
CL:0002195
CL:0002436
CL:0002453
CL:0002476
CL:0002664
CL:0004120
CL:0004138
CL:0005012
CL:0009106
CL:4023041
CL:4023097
CL:4023122
CL:4028002
CL:4028003
CL:4028004
CL:4028006

**Description of typo, bug or error**
In CL there are several synonyms that are (or include) abbreviations that are not annotated as "related" synonyms and / or are missing the 'abbreviation' annotation property tag. 
This ticket covers mostly synonyms currently annotated as narrow and broad. Given the volume of exact and related synonyms, a separate ticket will be planned.

**Fix**
Make synonyms that are abbreviations "related synonyms" and add 'abbreviation' synonym type.

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #1925 to add the tag abbreviation and changes to related synonyms

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -3808,10 +3808,10 @@ DisjointClasses(obo:CL_0000050 obo:CL_0002009)
 # Class: obo:CL_0000051 (common lymphoid progenitor)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:dsd") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:10407577") Annotation(oboInOwl:hasDbXref "PMID:16551251") obo:IAO_0000115 obo:CL_0000051 "A oligopotent progenitor cell committed to the lymphoid lineage.")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000051 "CLP")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000051 "common lymphocyte precursor")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000051 "common lymphocyte progenitor")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000051 "common lymphoid precursor")
+AnnotationAssertion(Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_0000051 "CLP")
 AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:CL_0000051 "ELP")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:3815]: Consider adding abbreviation tag to "ELP".

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -4602,12 +4602,12 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FB:ma") Annotation(oboInOwl:h
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000134 "BTO:0002625")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000134 "BTO:0003298")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000134 "FMA:70546")
-AnnotationAssertion(oboInOwl:hasNarrowSynonym obo:CL_0000134 "BMSC")
 AnnotationAssertion(oboInOwl:hasNarrowSynonym obo:CL_0000134 "bone marrow stromal cells")
 AnnotationAssertion(oboInOwl:hasNarrowSynonym obo:CL_0000134 "colony-forming unit-fibroblast")
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11378515") oboInOwl:hasNarrowSynonym obo:CL_0000134 "marrow stromal cells")
-AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:CL_0000134 "CFU-F")
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11378515") oboInOwl:hasRelatedSynonym obo:CL_0000134 "MSC")
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:CL_0000134 "BMSC")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:4608]: Consider adding abbreviation tag to "BMSC".

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -21080,9 +21080,9 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004120 "https
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004120 "https://orcid.org/0000-0002-5260-9315")
 AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0004120 "type I")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0004120 "BAMS:1045")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0004120 "I-2")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0004120 "inner alpha retinal ganglion cell")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0004120 "cell")
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:CL_0004120 "I-2")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:21085]: Consider adding abbreviation tag to "I-2".

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -21153,10 +21153,10 @@ SubClassOf(obo:CL_0004137 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0008925
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:12209831") obo:IAO_0000115 obo:CL_0004138 "A retinal ganglion A cell with dense arbor near soma.")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004138 "https://orcid.org/0000-0001-7258-9596")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004138 "https://orcid.org/0000-0002-5260-9315")
-AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_0004138 "type I")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0004138 "I-1")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0004138 "outer alpha retinal ganglion cell")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0004138 "cell")
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:CL_0004138 "I-1")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:21158]: Consider adding abbreviation tag to "I-1".

File: src/ontology/cl-edit.owl [hunk 5]
```diff
@@ -29395,12 +29395,12 @@ SubClassOf(obo:CL_4023040 obo:CL_4023008)
...
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.neuron.2005.08.030") oboInOwl:hasBroadSynonym obo:CL_4023041 "subcerebral projection (SCPN) neuron")
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.cell.2019.09.023") Annotation(rdfs:comment "pyramidal tract-like is used because, unlike pyramidal tract neurons in the motor cortex, neurons in the structures like the auditory cortex do not project to the spinal cord.") oboInOwl:hasExactSynonym obo:CL_4023041 "Pyramidal tract-like (PT-l)")
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.3389/fncel.2011.00001") oboInOwl:hasExactSynonym obo:CL_4023041 "burst-firing layer 5 neuron")
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.3389/fncel.2015.00233") oboInOwl:hasExactSynonym obo:CL_4023041 "thick-tufted layer 5 (TTL5) pyramidal neuron")
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1038/nrn3469") oboInOwl:hasNarrowSynonym obo:CL_4023041 "pyramidal tract (PT) neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/j.neuron.2011.07.029") oboInOwl:hasRelatedSynonym obo:CL_4023041 "L5b neuron")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:29403]: Consider adding abbreviation tag to "L5b neuron".

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
Please see inline comments.

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:21085]: In this case I was not sure if it was an abbreviation, as the label or the other sinonyms have nothing to do with it.

File: src/ontology/cl-edit.owl (hunk 4 above)
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:21158]: together with retinal ganglion cell A1, I will investigate them elsewhere. Both have as synonym Type I, and the related synonyms don't make sense.

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**